src/runtime/config.cpp

```diff
@@ -338,6 +338,13 @@ bool WakeConfig::init(const std::string& wakeroot_path, const WakeConfigOverride
   // Parse values from .wakeroot
   _config->set_all<WakeConfigProvenance::WakeRoot>(wakeroot_json);
 
+  // Sometimes we need to the user_config with an env-var so we check env-vars first here
+  _config->set_all_env_var();
```

Comment:
The priority of each source is a bit unclear to me. Can we document that directly somewhere?